config/crd/serving.kserve.io_inferenceservices.yaml

```diff
@@ -4943,6 +4943,8 @@ spec:
                           type: array
                         framework:
                           properties:
+                            autoPlace:
```

Comment:
@pvaneck I think it is better to keep the names consistent between SR and InferenceService, I prefer `Framework` as `ModelType` in ML world often means binary/multiclass classifier or regression.

Reply:
I guess we should then rename the field in SR to be something like `supportedModelFrameworks`?

Just throwing it out there but how about `ModelFormat` / `supportedModelFormats`?

Reply:
I am fine with either `supporedModelFrameworks` or `supportedModelFormats`. Do note, that this will require a small followup PR in ModelMesh-Serving to adjust the SR spec there as well. (Eventually, we'll consolidate to a single source)

Reply:
Now I think `ModelFormat` might make more sense as we have `onnx`, `pmml`, `tensorrt` on the list..